pyvista/core/objects.py

```diff
@@ -375,6 +375,24 @@ def repeat(self):
     def repeat(self, flag):
         self.SetRepeat(flag)
 
+    @property
+    def cube_map(self, flag):
+        """Is this texture a cube map, if so it needs 6 inputs one for each
+        side of the cube. You must set this before connecting the inputs.
+        The inputs must all have the same size, data type, and depth.
+        """
+        return self.GetCubeMap()
+
+    @repeat.setter
```

Comment:
This should be `cube_map.setter` instead, I believe.

Reply:
About to be fixed in 634da875017386f367befa0dab739dff9cb8ea09

Reply:
You haven't pushed this commit yet @akaszynski, right? I can't see it.